examples/kafka_pubsub/go.mod

```diff
@@ -1,5 +1,7 @@
 module github.com/TykTechnologies/graphql-go-tools/examples/kafka_pubsub
 
-go 1.15
+go 1.19
 
-require github.com/Shopify/sarama v1.29.1
+require (
+	github.com/IBM/sarama v1.14.0 // indirect
```

Comment:
Is that a typo? We can upgrade `github.com/IBM/sarama` from `v1.14.0` to `1.41.0` here. 1.14 is too old.

Reply:
Hmm no, i simply ran the go get sarama@latest, i don't recall specifying the version, i will check this again